webrtc/libjingle/xmpp/xmppengineimpl.h

```diff
@@ -247,6 +273,8 @@ class XmppEngineImpl : public XmppEngine {
   XmppOutputHandler* output_handler_;
   XmppSessionHandler* session_handler_;
 
+  XmppStanzaGenerator* content_wrapper_;
```

Comment:
I think we should change content_wrapper_ to stanza_generator_ or something (after we changed the class name, this older name doesn't make much sense)